src/test/java/org/jnode/fs/xfs/XfsFileSystemTest.java

```diff
@@ -180,10 +180,10 @@ public void testCentos() throws Exception {
         try (FileDevice device = new FileDevice(testFile, "r")) {
             XfsFileSystemType type = new XfsFileSystemType();
             XfsFileSystem fs = type.create(device, true);
-            FSEntry entry = fs.getRootEntry();
+            XfsEntry entry = fs.getRootEntry();
```

Comment:
I would've thought that the return value from `fs.getRootEntry()` would needed to be cast to `XfsEntry` like it is in `testXfsMetaData()`.

Reply:
Never mind. I can see why it doesn't need to be cast.

Reply:
HI , Erwin, was not necessary to cast to XfsEntry. Thanks